### ISSUE TITLE
Exterminate! Exterminate! Should let Player choose order of usage.

### ISSUE THREAD

**Issue by Achocholacek** [keyteki/keyteki#200]
Right now, when you play Exterminate! Exterminate! - the order of creature destruction is based on the position of the mars creature in your battleline.

It should allow for order selection to make sure you are not forced in to sub-optimal play, and the messaging should be better.

**Comment by cryogen** [comment]
Is this not another case of needing to have the prompt for order of triggered enabled?

**Comment by Achocholacek** [comment]
The option does not effect EE - but perhaps EE can be written so it will.

**Comment by hydrophilic-attack** [comment]
Does the order in which you choose targets matter? The destruction is likely supposed to be simultaneous, like 3 fates

**Comment by Achocholacek** [comment]
That is a fair question. Thoughts from anyone else?

**Comment by cryogen** [comment]
Yeah that's a fair point, if the damage is simultaneous, the order you pick is not important.

**Comment by hydrophilic-attack** [comment]
The thing is, if it's simultaneous, you'd still need to change how the card works a bit.

Because if it's simultaneous, you should be able to select the same creature to be destroyed multiple times, similar to how other "for each x, do y to a creature" effects work. (It'd still only be destroyed once.) Because the creature you are selecting over and over wouldn't have left play until the effect procs.

**Comment by cryogen** [comment]
The new timing chart and destroyed clarifications don't allow a creature to be tagged multiple times.  Can we close this now?

**Comment by hydrophilic-attack** [comment]
A rules question email actually gave us an answer for how this card is supposed to work

Look at the following link:

https://archonarcana.com/index.php?title=Exterminate!_Exterminate!

To summarize: You must choose a different creature each time. You don't have to maximize the number of creatures you destroy. (You can choose your mars creatures in such an order that not all friendly mars creature matches up with a lower power non-mars creature in order to spare creatures you'd want to stick around.) All chosen creatures are destroyed simultaneously (which matters for cards like Archimedes, Lion Bautrem, Haedroth's Wall).

Actually, I think this ruling doesn't quite line up with th rules as written, but this is what they ruled. It may be an indication of an upcoming errata or rules change.

If you want to change the way it works here, I'd make it so that you first get to click one of your Mars creatures, then a non-Mars creature with lower power, and repeat this until all friendly Mars creatures have been selected.
